extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/CircuitHandler.java

```diff
@@ -59,7 +59,7 @@
     /**
      * Contains all supported thing types of this handler, will be filled by DsDeviceThingTypeProvider.
      */
-    public static Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<ThingTypeUID>();
+    public static Set<ThingTypeUID> circuitHandlerSupportedThingTypes = new HashSet<ThingTypeUID>();
```

Comment:
this is also a constant: No other entity is allowed to change the reference. So: `public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES` please.